src/pages/home/ReportScreen.js

```diff
@@ -204,7 +204,9 @@ class ReportScreen extends React.Component {
         // We let Free Plan default rooms to be shown in the App - it's the one exception to the beta, otherwise do not show policy rooms in product
         if (!Permissions.canUseDefaultRooms(this.props.betas)
             && ReportUtils.isDefaultRoom(this.props.report)
-            && ReportUtils.getPolicyType(this.props.report, this.props.policies) !== CONST.POLICY.TYPE.FREE) {
+            && ReportUtils.getPolicyType(this.props.report, this.props.policies) !== CONST.POLICY.TYPE.FREE
+            && ReportUtils.hasExpensifyGuidesEmails(lodashGet(this.props.report, ['participants'], []))
```

Comment:
This should be a `!` right? This is filtering out reports, not checking if we can add them